flink/flink-scala-2.12/pom.xml

```diff
@@ -904,31 +1332,5 @@
         </dependency>
       </dependencies>
     </profile>
-
-    <profile>
-      <id>hive2</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
-      <properties>
-        <hive.version>2.3.4</hive.version>
-        <hiverunner.version>4.0.0</hiverunner.version>
-      </properties>
-    </profile>
-    <profile>
-      <id>hive1</id>
```

Comment:
Hive 1.x is officially dead https://lists.apache.org/thread/c7lyv2c2phzyspov48yxjnvks944tmkd, nor support Hadoop 3 neither